test/e2e/framework/metrics/metrics_grabber.go

```diff
@@ -151,7 +161,7 @@ func (g *Grabber) GrabFromScheduler() (SchedulerMetrics, error) {
 
 // GrabFromClusterAutoscaler returns metrics from cluster autoscaler
 func (g *Grabber) GrabFromClusterAutoscaler() (ClusterAutoscalerMetrics, error) {
-	if !g.registeredMaster && g.externalClient == nil {
+	if !g.HasControlPlanePods() && g.externalClient == nil {
 		return ClusterAutoscalerMetrics{}, fmt.Errorf("Master's Kubelet is not registered. Skipping ClusterAutoscaler's metrics gathering")
```

Comment:
i guess we need to replace:
> "Master's Kubelet is not registered. Skipping ClusterAutoscaler's metrics gathering"

with something like:
> Did not find control-plane pods. Skipping ClusterAutoscaler's metrics gathering

"master's kubelet" was misleading originally too.

Reply:
Thanks again, `Did not find control-plane pods. ` will be applied.